src/components/Header/index.js

```diff
@@ -31,7 +31,7 @@ const Header = ({
             </LogoContainer>
           </Col>
           <Col grow={false} onClick={setLgn}>
-            <LocaleContainer>
+            <LocaleContainer color="white" textSize="0.875rem" weight="500">
```

Comment:
Both `LogoContainer` and `LocaleContainer` will need to have their `margin` set to zero since they now inherit a `margin-bottom` of 6px and the `Header` doesn't look aligned